### ISSUE TITLE
Reddit subreddit blocking broken

### ISSUE THREAD

**Issue by tinaLa** [tobiasdalhof/sanersocialmedia#40]
Hi, great plugin! Really appreciate the ability to block additional pages beyond the homepage. However, I noticed the subreddit feed option doesn't appear to be working for me.

**Comment by tobiasdalhof** [comment]
It works for me, is "block subreddit feeds" enabled in your site settings?

![chrome_VBGP5KIYRL](https://github.com/tobiasdalhof/sanersocialmedia/assets/10046578/36faf1a6-cd1b-47d2-a355-aff99979dc79)

**Comment by tinaLa** [comment]
Yes, it is. Here's a screen recording demonstrating the problem.

https://github.com/tobiasdalhof/sanersocialmedia/assets/22090204/28c9887b-73dc-4145-8d29-3ceae0a48736

**Comment by tobiasdalhof** [comment]
Thank you! I recently updated the extension to only work with the new Reddit design. I was under the impression that Reddit now displays the new design for all its users...

**Comment by tinaLa** [comment]
I believe this is the new design, I just have the sidebar collapsed. Unless I'm missing something here? 

Logging out brings up a similar layout to what you shared so the extension works there but not when I'm logged in.

**Logged Out (Working)**
![Screen Shot 2024-04-09 at 8 17 06 AM](https://github.com/tobiasdalhof/sanersocialmedia/assets/22090204/708937d0-4643-467f-8b4c-80630cd9c162)

**Logged In (Not Working)**
![Screen Shot 2024-04-09 at 8 19 02 AM](https://github.com/tobiasdalhof/sanersocialmedia/assets/22090204/3077869d-8087-4b8f-8be9-6c1cbe94b2a0)

**Comment by tobiasdalhof** [comment]
Logged in screenshot is using the old design, check this:

![image](https://github.com/tobiasdalhof/sanersocialmedia/assets/10046578/c6463547-bf55-4616-a89a-b37f9c6b1d8c)

**Comment by tinaLa** [comment]
Ah okay, I did some sleuthing and looks like this is a "new new" redesign that's only accessible for me through sh.reddit.com. I opted in for beta features in my settings to see if it'll give my account the new layout but seems like it may not kick in immediately.

Either way, it seems like this "new new" redesign hasn't completely rolled out to all users so I'll leave that to your consideration. Would love to have the subreddit blocking feature supported on the "old new" design but I understand if there was an important reason for why it is not.
